src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -285,10 +286,19 @@ bool emberAfGroupKeyManagementClusterKeySetWriteCallback(
     chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
     const chip::app::Clusters::GroupKeyManagement::Commands::KeySetWrite::DecodableType & commandData)
 {
-    auto fabric     = commandObj->GetAccessingFabricIndex();
-    auto * provider = GetGroupDataProvider();
+    auto provider = GetGroupDataProvider();
+    auto fabric   = Server::GetInstance().GetFabricTable().FindFabricWithIndex(commandObj->GetAccessingFabricIndex());
```

Comment:
How will this work on a controller without a Server Instance ?

Reply:
I would assume that without a server instance, you won't be managing keys with in-band Matter communication in the first place.  No?

Reply:
Correct. This files handles the server side of the exchange. This code won't execute in a client-only application.